x-pack/plugins/ml/public/application/jobs/jobs_list/components/datafeed_modal/datafeed_modal.tsx

```diff
@@ -298,7 +302,18 @@ export const DatafeedModal: FC<DatafeedModalProps> = ({ jobId, end, onClose }) =
                         })}
                         position={Position.Left}
                       />
+                      {showAnnotations ? (
+                        <RectAnnotation
```

Comment:
The chart only seems to display the annotation if the end time is different to the start time, so most of the ones generated by the back-end aren't displayed. Is there any way of handling this in Elastic charts? Otherwise we might want to add some padding to the start / end data used by the annoation series.

Reply:
Added chart LineAnnotation to handle those cases in 645f1468dd80cf8ce7fb644e84ced42fb03f9699

![image](https://user-images.githubusercontent.com/6446462/122263448-70f51380-cea4-11eb-809b-e00fb2c04863.png)